src/main/java/com/unosquare/admin_core/back_end/repository/DashboardRepository.java

```diff
@@ -12,20 +12,21 @@
 
 public interface DashboardRepository extends JpaRepository<Event, Integer> {
     @Query(value = "SELECT e FROM Event e " +
-            "WHERE " +
+            "WHERE NOT( e.event_status_id = '4' AND e.last_modified < NOW()- interval '36' hour) " +
+            "AND " +
```

Comment:
Why are we using native sql here? We should be using hsql. The only time native should be used is if it's not possible to do the logic in hsql. I can't see anything here that wouldn't be allowed.

Reply:
Any use of 'current_timestamp', 'DATEADD(hour, -36, GETDATE())', 'GETDATE()' or 'NOW()' resulted in an error with the custom query. Using 'nativeQuery = true' got around this for the query to work. I am happy to insert any changes that can remove the native query option and run the clause as expected. That's why nativeQuery is present in these queries.

Reply:
You could pass this in as a variable then? Or there must be some way of getting today's date using hsql?

Reply:
I've made that change and passed in the variable we need. No longer using nativeQuery in the repo. Cheers Eoin. That's it pushed up on to the branch.